COGS/automod.py

```diff
@@ -606,7 +606,7 @@ def is_excluded(match, exclusions):
                 async def run_delete(msg):
                     try:
                         await msg.delete()
-                    except:
+                    except Exception as e:
```

Comment:
Why? If you aren't going to use the exception, don't capture it. It might use more resources if you do.

Reply:
It's to debug as messages aren't deleted, will remove eventually. Besides resource usage is minimal.